### PR TITLE
Add leaderboard to all Mujoco pbs

### PR DESCRIPTION
**Pull request by teytaud**
This PR adds the Mujoco environments in the leaderboard.
Maybe we should do more than that: check that the value is preserved in the unit test (i.e. at least -4000 for humanoid, etc).

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] The documentation is up-to-date with the changes I made.
- [x] I have read the [**CONTRIBUTING**](https://facebookresearch.github.io/nevergrad/contributing.html) document and completed the CLA (see [**CLA**](https://facebookresearch.github.io/nevergrad/contributing.html#contributor-license-agreement-cla)).
- [x] All tests passed, and additional code has been covered with new tests.

### CODE REVIEW

File: nevergrad/functions/control/core.py [hunk 1]
```diff
@@ -68,6 +69,14 @@ def state_std(self):
     @property
     def policy_dim(self):
         raise NotImplementedError
+        
+    # pylint: disable=arguments-differ
+    def evaluation_function(self, x: np.ndarray) -> float:  # type: ignore
+        # pylint: disable=not-callable
+        loss = self.function(x)
```
Comment (jrapin) [inline nevergrad/functions/control/core.py:76]: is there no noise in mujoco? if there is noise, it may be worth averaging over several runs.

Comment (teytaud) [reply nevergrad/functions/control/core.py:76]: The noise is not merged, and I do not expect it to be merged in the near future.

Comment (jrapin) [reply nevergrad/functions/control/core.py:76]: I mean, when there is no added noise, the behavior of the function is deterministic then? If yes that's fine and you can merge then

Comment (teytaud) [reply nevergrad/functions/control/core.py:76]: Yes!

**Pull request merged**